### PR TITLE
support @PrepareEverythingForTest in testng and junit, @PrepareOnlyThisForTest in testng

### PR DESCRIPTION
**Pull request by astafev**
Not sure how to act with tests, because they are practically the same, but with different annotations.

### CODE REVIEW

**Comment by thekingnothing** [comment]
Test failed: 

Tests in error: 
  PrepareEverythingForTestTest.testMockStaticNoExpectations:36 NoSuchMethod org....

File: core/src/main/java/org/powermock/core/classloader/MockClassLoader.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
             "net.sf.cglib.", "javassist.",
             "org.powermock.modules.junit4.internal.", "org.powermock.modules.junit4.legacy.internal.",
             "org.powermock.modules.junit3.internal.",
-            "org.powermock"};
+            "org.powermock", "org.mockito"};
```
Comment (thekingnothing) [inline core/src/main/java/org/powermock/core/classloader/MockClassLoader.java]: I'm not sure that "org,mockito" should be always loaded via MockClass loader

File: core/src/main/java/org/powermock/core/classloader/MockClassLoader.java [hunk 2]
```diff
@@ -297,6 +297,7 @@ protected boolean shouldLoadUnmodifiedClass(String className) {
     }
 
     private boolean shouldLoadWithMockClassloaderWithoutModifications(String className) {
+        //
```
Comment (thekingnothing) [inline core/src/main/java/org/powermock/core/classloader/MockClassLoader.java]: Unnecessary comment

**Comment by astafev** [comment]
sorry for not answering for so long time, I probably won't be able to make this PR up to date with last changes. Don't know if adding "org.mocktio" to exclusion will break anything, but @PrepareEverythingForTest wasn't working in very simple use case. Please take a look at this annotation if you have time. I'm closing PR as I don't know how to merge it now.

**Pull request closed without merging**